API/ev3_sensors/utility.c

```diff
@@ -0,0 +1,12 @@
+#include "API/ev3_inputs/ev3_input_uart.h"
+#include "include/ev3_command.h"
+#include "utility.h"
+
+void setUARTSensorHandlerMode(SensorHandler *handler, int port, int type, int mode) {
+    int previousMode = handler->currentSensorMode[port];
+    setUARTSensorMode(port, type, mode);
+    EV3Color->currentSensorMode[port] = mode;
```

Comment:
This should probably be `handler->...`, I will submit a fix for it.

Reply:
Fix available in https://github.com/simonedegiacomi/EV3-API/commit/5faf594b5ff272317cd3751c089c0730013d262a